code/modules/reagents/Chemistry-Colours.dm

```diff
@@ -12,6 +12,8 @@
 	var/vol_temp
 	// see libs/IconProcs/IconProcs.dm
 	for(var/datum/reagent/reagent in reagent_list)
+		if (pigments_only && !(reagent.flags & CHEMFLAG_PIGMENT))
+			continue
 		if(reagent.id == BLOOD && reagent.data["blood_colour"])
 			reagent_color = reagent.data["blood_colour"]
```

Comment:
sort of related, the color blending a couple lines down is wrong - mixing blue and yellow paint makes grey instead of green - IRL paints don't mix by just adding the RGB values
this is probably a pretty complicated thing to code, though

Reply:
You'll never guess what I've been discussing in the Discord's coder chat for almost the past 18 hours.

My conclusions so far are that colours mixing is a cursed science and IRL pigments don't care about maths.

I really want my next big PR to be something paints related now, and standardize color mixing for both paints and reagents, but we'll see how far I get.

Reply:
Do not research IRL pigment science you will lose your mind.